src/commands/check.rs

```diff
@@ -119,9 +124,61 @@ pub async fn check(args: CheckArgs) -> anyhow::Result<()> {
         );
     }
 
-    let remote_file = Url::parse(&file).is_ok();
+    let remote_file = Url::parse(file).is_ok();
+
+    if !exceptions.is_empty() {
+        let analysis_rules: HashSet<String> = analysis::rules()
+            .iter()
+            .map(|r| r.id().to_lowercase())
+            .collect();
+        let lint_rules: HashSet<String> = lint::rules()
+            .iter()
+            .map(|r| r.id().to_lowercase())
+            .collect();
+        let all_rules: HashSet<_> = analysis_rules.union(&lint_rules).collect();
```

Comment:
This misses `ShellCheck`! Then again, as stated in #91 the `optional_rules()` is going away in the next release, so it might be ok to ignore for now.